README.md

```diff
@@ -170,7 +170,13 @@ If you do not have the community repo configured in your `KISS_PATH`, follow the
 
 `glab` is available on the [Alpine Community Repo](https://git.alpinelinux.org/aports/tree/community/glab?h=master) as `glab`.
 
-**Warning:** The package is currently only available under the `edge`. `edge` is under constant development so be careful using it in production. It is possible that bugs in `edge` could cause data loss or could break your system.
+##### Install
+
+We use `--no-cache` so we don't need to do an `apk update` before.
+
+```sh
+apk add --no-cache glab
```

Comment:
`apk add -U`

Reply:
I would let with `--no-cache` it's better for understanding that have not so much experience.